### PR TITLE
[release]Linux/SQL 

### PR DESCRIPTION
**Pull request by AdityaSharma5**

### CODE REVIEW

File: linux_sql/scripts/psql_docker.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+#script pseudocode
+#hint: `systemctl status docker || ...`
+#if docker_deamon is not running
+#	start docker_deamon
```
Comment (jarviscanada) [inline linux_sql/scripts/psql_docker.sh]: * Remove unused comments
* Please review shell coding style https://www.notion.so/jarviscanada/Shell-Style-Guide-764a3ac0c9044fdb844b55420d1c7594

File: linux_sql/scripts/psql_docker.sh [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+if [ $docker_command == "start" ]; then
+  docker container start jrvs-psql
+  exit 0
+fi
+if [ $docker_command == "stop" ];then
+  docker container stop jrvs-psql
+  exit 0
+fi
+
+if [[ $docker_command != "create"  ||  $docker_command != "start"  ||   $docker_command != "stop" ]];then
```
Comment (jarviscanada) [inline linux_sql/scripts/psql_docker.sh]: this if is not necessary as you already checked all actions before

File: linux_sql/scripts/psql_docker.sh (hunk 1 above)
Comment (jarviscanada) [reply linux_sql/scripts/psql_docker.sh]: remove all hints and leave necessary comments only

File: linux_sql/scripts/psql_docker.sh [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  #What's $? variable? https://bit.ly/2LanHUi
+  exit 0
+fi
+
+if [[ $(docker container ls -a -f name=jrvs-psql) == "" ]];then
+  echo "jarvs-psql not created"
+  exit 1
+fi
+
+if [ $docker_command == "start" ]; then
```
Comment (jarviscanada) [inline linux_sql/scripts/psql_docker.sh:39]: check container existence

File: linux_sql/scripts/psql_docker.sh [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  docker container start jrvs-psql
+  exit 0
```
Comment (jarviscanada) [inline linux_sql/scripts/psql_docker.sh]: use `exit $?`

**Review by jarviscanada — APPROVED** [review]
fix all and merge

**Review by jarviscanada — CHANGES_REQUESTED** [review]
fix and re-open RCR from 360 page

File: linux_sql/scripts/psql_docker.sh (hunk 2 above)
Comment (AdityaSharma5) [reply linux_sql/scripts/psql_docker.sh]: fixed

File: linux_sql/scripts/psql_docker.sh (hunk 3 above)
Comment (AdityaSharma5) [reply linux_sql/scripts/psql_docker.sh:39]: fixed

File: linux_sql/scripts/psql_docker.sh (hunk 4 above)
Comment (AdityaSharma5) [reply linux_sql/scripts/psql_docker.sh]: fixed

**Pull request merged**